models/Post.php

```diff
@@ -132,7 +136,9 @@ public function scopeListFrontEnd($query, $options)
          * Categories
          */
         if ($categories !== null) {
-            if (!is_array($categories)) $categories = [$categories];
+            $categories = Category::find($categories);
```

Comment:
No more support for multiple categories... Perhaps we should leave "categories" intact and add a new option called "category" that does this instead.